app/serializers/season_serializer.rb

```diff
@@ -3,8 +3,8 @@
 # Serializes the season to JSON
 class SeasonSerializer < Oj::Serializer
   attributes :id, :name, :season_number, :episode_count, :slug
-  serializer_attributes :poster_url
 
+  attribute \
   def poster_url
     season.poster.url
   end
```

Comment:
This is unrelated, just wanted to mention that you can do this in Oj Serializers 😄